test/gurka/test_search_filter.cc

```diff
@@ -584,17 +584,19 @@ TEST_P(ExcludeClosuresOnWaypoints, DISABLED_ConsecutiveClosuresWithLowReachabili
 
     // Specify minimum reachability so that nearby edges (which were previously
     // rejected due to low-reachbility), can be selected
+    // Note: we must turn off candidate ranking because of a bug in bidira* that
+    // doesnt continue the other direction of the search if one becomes exhausted
     const std::string& req_low_reachbility =
         (boost::format(
-             R"({"locations":[{"lat":%s,"lon":%s,"search_filter":{"exclude_closures":false}, "minimum_reachability": 5},{"lat":%s,"lon":%s}],"costing":"%s", "costing_options": {"%s": {"speed_types":["freeflow","constrained","predicted","current"]}}, "date_time":{"type":"%s", "value": "current"}})") %
+             R"({"locations":[{"lat":%s,"lon":%s,"search_filter":{"exclude_closures":false}, "minimum_reachability": 2, "rank_candidates":false},{"lat":%s,"lon":%s}],"costing":"%s", "costing_options": {"%s": {"speed_types":["freeflow","constrained","predicted","current"]}}, "date_time":{"type":"%s", "value": "current"}})") %
```

Comment:
the problem we were encountering was that the reach was wrong when we tell the library to ignore closures for locations. in this case the reach computation would end up thinking that an edge behind a multi edge closure was reachable because it uses the, previusly named, `Filter` function, to do the quick conservative reach measurement. The problem is this conservative check was no longer conservative because we bundled the exclude closure flag into the `Filter` function. That was a mistake because it made locations that werent reachable look like they were. And would therefor fail the route instead of snapping somewhere else

Reply:
also note we need to turn of `rank_candidates` here because of a bug in bidirectional a* where it runs out of stuff in one of the search branches but doesnt continue in the other even if it has stuff